backend/evidence/src/main/java/org/pucar/dristi/enrichment/EvidenceEnrichment.java

```diff
@@ -110,7 +110,7 @@ public void enrichEvidenceNumber(EvidenceRequest evidenceRequest) {
                     1,
                     false
             );
-            evidenceRequest.getArtifact().setEvidenceNumber(evidenceNumberList.get(0));
+            evidenceRequest.getArtifact().setEvidenceNumber(evidenceRequest.getArtifact().getCnrNumber()+"-"+evidenceNumberList.get(0));
             evidenceRequest.getArtifact().setIsEvidence(true);
```

Comment:
Evidence/Artifact Registry has two fields 

artifactNumber and evidenceNumber. 

When an artifact is initially submitted it will get an artifact number. Once accepted as evidence, the artifact number will be copied to evidence number. It will **not** be a new number. in addition isEvidence will be set to TRUE. 

when artifact is unmarked as evidence, the evidence number field will be set to blank and the isEvidence will be set to FALSE.

Reply:
@atulgupta2024 
 In evidence we are generating evidenceNumber based on artifactType_sourceType mapping like this. can you validate below for different idName
static {
    artifactSourceMap.put("DOCUMENTARY_COMPLAINANT", "case.evidence.prosecution.[TENANT_ID]");
    artifactSourceMap.put("DOCUMENTARY_ACCUSED", "case.evidence.defence.[TENANT_ID]");
    artifactSourceMap.put("DOCUMENTARY_COURT", "case.evidence.court.[TENANT_ID]");
    artifactSourceMap.put("AFFIDAVIT_COMPLAINANT", "case.evidence.prosecution.[TENANT_ID]");
    artifactSourceMap.put("AFFIDAVIT_ACCUSED", "case.evidence.defence.[TENANT_ID]");
    artifactSourceMap.put("AFFIDAVIT_COURT", "case.evidence.court.[TENANT_ID]");
    artifactSourceMap.put("DEPOSITION_COMPLAINANT", "case.evidence.prosecution.witness.[TENANT_ID]");
    artifactSourceMap.put("DEPOSITION_ACCUSED", "case.evidence.defence.witness.[TENANT_ID]");
    artifactSourceMap.put("DEPOSITION_COURT", "case.evidence.court.witness.[TENANT_ID]");
}

Is this mapping correct cc : @manimaarans